lib/management/types.ts

```diff
@@ -391,6 +393,17 @@ export type UserPasswordDjango = {
   hash: string; // the django hash in plaintext format, for example "pbkdf2_sha256$..."
 };
 
+export type UserPasswordPhpass = {
```

Comment:
following my comment on the readme I think this is the only type that is exported, see for example the user mapping

![image](https://github.com/user-attachments/assets/d042710d-0b41-4d6c-9fad-84b4ee277697)

![image](https://github.com/user-attachments/assets/2b488208-fead-4082-902c-cfcc2f00648a)

also, specifically for this `UserPasswordPhpass` and `UserPasswordMd5`, are they used anywhere?

Reply:
yeah, they're used indirectly in `inviteBatch`

Reply:
What do you mean by that?